### PR TITLE
[CHORE] Add encrypt-key command to CLI

### PR DESCRIPTION
**Pull request by mmrozek**
# Description

`encrypt-key` command could be useful to set up a faucet

### CODE REVIEW

File: src/main/scala/io/iohk/ethereum/cli/CliCommands.scala [hunk 1]
```diff
@@ -74,6 +78,23 @@ object CliCommands extends SecureRandomBuilder {
       }
     }
 
+  private val EncryptKey: Command[String] =
+    Command(name = encryptKeyCommand, header = "Encrypt private key") {
+
+      val privateKey = Opts
```
Comment (kapke) [inline src/main/scala/io/iohk/ethereum/cli/CliCommands.scala]: Shouldn't common options be defined outside commands? Or maybe even all options should replace current string constants in lines ~17-25?

File: src/test/scala/io/iohk/ethereum/cli/CliCommandsSpec.scala [hunk 2]
```diff
@@ -92,6 +94,30 @@ class CliCommandsSpec extends AnyFlatSpec with Matchers with EitherValues {
...
+
+    val decrypted = (for {
+      encrypted <- EncryptedKeyJsonCodec.fromJson(json)
+      decrypted <- encrypted.decrypt("")
+    } yield decrypted).value
+
+    ByteStringUtils.hash2string(decrypted) shouldBe privateKey
+  }
+
+  it should "encrypt private key (with passphrase)" in {
```
Comment (kapke) [inline src/test/scala/io/iohk/ethereum/cli/CliCommandsSpec.scala:110]: minor: this test looks like a perfect candidate for a property-based one.

File: README.md [hunk 3]
```diff
@@ -50,6 +50,11 @@ Possible networks: `etc`, `eth`, `mordor`, `testnet-internal`
 ./bin/mantis cli generate-key-pairs 5
 ```
 
+- encrypt private key (could be used to setup private faucet)
```
Comment (kapke) [inline README.md]: Probably bit more documentation (and explanation) would be helpful. Either here or in the help for that command

File: src/test/scala/io/iohk/ethereum/cli/CliCommandsSpec.scala (hunk 2 above)
Comment (mmrozek) [reply src/test/scala/io/iohk/ethereum/cli/CliCommandsSpec.scala:110]: Not really. I don't want to test decryption/encryption here. It should be done in a different place. I only want to check if I get the correct JSON when I specify the correct arguments

File: src/main/scala/io/iohk/ethereum/cli/CliCommands.scala (hunk 1 above)
Comment (mmrozek) [reply src/main/scala/io/iohk/ethereum/cli/CliCommands.scala]: I prefer to leave string constants because it is convenient to use them in tests

**Review by biandratti — APPROVED** [review]

**Review by lemastero — APPROVED** [review]
LGTM 👍

**Review by kapke — APPROVED** [review]
LGTM!

**Pull request merged**